server/sonar-qa-util/src/main/java/org/sonarqube/qa/util/UserTester.java

```diff
@@ -105,23 +106,23 @@ public final User generateAdministratorOnDefaultOrganization(Consumer<CreateRequ
   @SafeVarargs
   public final User generateMember(Organizations.Organization organization, Consumer<CreateRequest.Builder>... populators) {
     User user = generate(populators);
-    session.wsClient().organizations().addMember(organization.getKey(), user.getLogin());
+    session.wsClient().organizations().addMember(new AddMemberRequest().setOrganization(organization.getKey()).setLogin(user.getLogin()));
     return user;
   }
 
   @SafeVarargs
   public final User generateMemberOfDefaultOrganization(Consumer<CreateRequest.Builder>... populators) {
     User user = generate(populators);
-    session.wsClient().organizations().addMember(DEFAULT_ORGANIZATION_KEY, user.getLogin());
+    session.wsClient().organizations().addMember(new AddMemberRequest().setOrganization(DEFAULT_ORGANIZATION_KEY).setLogin(user.getLogin()));
     return user;
   }
 
   public UsersService service() {
-    return session.wsClient().users();
+    return session.wsClient().usersOld();
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove this use of "usersOld"; it is deprecated. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3ACallToDeprecatedMethod)